### DISCUSSION TITLE
Recurring Costs

### DISCUSSION THREAD

**Discussion by jens-2305** [kimai/kimai#3305] [Questions & Answers]
Hi,

is it possible to configure recurring costs, e.g. monthly maintenance costs or licence costs every 12 months or similar ? For a customer or a project

Thank you very much

**Comment by kevinpapst** [comment]
AFAIK there is no plugin available for that. 
But it is on my (very long) todo list for the expense plugin.

**Comment by charismeki** [comment]
Hi Kevin and Jens, exactly what we are looking for as well :-) Kind regards, c--

**Comment by remcovg89** [comment]
We have the same requirement, is there any workaround?
